ee/ucp/admin/configure/use-your-own-tls-certificates.md

```diff
@@ -30,33 +30,27 @@ will have to download new ones to [access UCP from the CLI](../../user-access/cl
 
 ## Configure UCP to use your own TLS certificates and keys
 
-In the UCP web UI, log in with administrator credentials and
-navigate to the **Admin Settings** page.
+To configure UCP to use your own TLS certificates and keys:
 
-  In the left pane, click **Certificates**.
+1. In the UCP web UI, log in with administrator credentials and
+navigate to the **Admin Settings** page.
 
-![](../../images/use-externally-signed-certs-2.png)
+2. Click **Certificates**.
 
-Upload your certificates and keys:
+    ![](../../images/ucp-certs.png)
+    
+3. Upload your certificates and keys:
+* **Private key:** The unencrypted private key of UCP. This key must correspond to the public key used in the server certificate. Click **Upload Key**.
+* **Server certificate:** The public key certificate of UCP followed by the certificates of any intermediate certificate authorities which establishes a chain of trust up to the root CA certificate. Click **Upload Certificate** to upload a PEM file.
+* **CA certificate:** The public key certificate of the root certificate authority that issued the UCP server certificate. If you don't have one, use the top-most intermediate certificate instead. Click **Upload CA Certificate** to upload a PEM file.
+* **Client CA:** This field is available in UCP 3.2. This field may contain one or more Root CA certificates which the UCP Controller will use to verify that client certificates are issued by a trusted entity. UCP is automatically configured to trust its internal CAs which issue client certificates as part of generated client bundles, however, you may supply UCP with additional custom root CA certificates here so that UCP may trust client certificates issued by your corporate or trusted third-party certificate authorities. Note that your custom root certificates will be appended to UCP's internal root CA certificates. Click **Upload CA Certificate** to upload a PEM file. Click **Download UCP Server CA Certificate** to download the certificate as a PEM file.
 
```

Comment:
Are we averse to using tables? In this case and others like it, I would normally default to a simple table without a Heading row.